contracts/dev/deploy-testnet

```diff
@@ -23,4 +23,7 @@ function deploy_contract() {
 
 deploy_contract src/GroupMessages.sol GroupMessages
 deploy_contract src/Nodes.sol Nodes
-deploy_contract src/IdentityUpdates.sol IdentityUpdates
\ No newline at end of file
+deploy_contract src/IdentityUpdates.sol IdentityUpdates
+
+# TODO: Migrate to forge script.
```

Comment:
Hopefully the forge script also fixes the flakiness I've been seeing with contract verification.